packages/patternfly-3/react-console/src/VncConsole/VncActions.js

```diff
@@ -1,30 +1,59 @@
 import React from 'react';
+import ReactDOM from 'react-dom';
 import PropTypes from 'prop-types';
 import { MenuItem, Button, helpers } from 'patternfly-react';
 
 const { Dropdown } = Button;
 const { noop } = helpers;
 
-const VncActions = ({ textSendShortcut, textCtrlAltDel, onCtrlAltDel }) => (
-  <Dropdown bsStyle="default" title={textSendShortcut} id="console-send-shortcut" onClick={noop}>
-    <MenuItem eventKey="1" onClick={onCtrlAltDel}>
-      {textCtrlAltDel}
-    </MenuItem>
-  </Dropdown>
-);
+const VncActions = ({
+  textSendShortcut,
+  textCtrlAltDel,
+  textDisconnect,
+  onCtrlAltDel,
+  onDisconnect,
+  insertToolbarTo
+}) => {
+  const toolbar = (
+    <div>
+      <Dropdown bsStyle="default" title={textSendShortcut} id="console-send-shortcut" onClick={noop}>
+        <MenuItem eventKey="1" onClick={onCtrlAltDel}>
+          {textCtrlAltDel}
+        </MenuItem>
+      </Dropdown>
+      <Button bsStyle="default" onClick={onDisconnect}>
+        {textDisconnect}
+      </Button>
+    </div>
+  );
+
+  if (!insertToolbarTo) {
+    return toolbar;
+  }
+  return (
+    document.getElementById(insertToolbarTo) && ReactDOM.createPortal(toolbar, document.getElementById(insertToolbarTo))
+  );
+};
 
 VncActions.propTypes = {
+  onDisconnect: PropTypes.func.isRequired,
   onCtrlAltDel: PropTypes.func,
 
   textCtrlAltDel: PropTypes.string,
-  textSendShortcut: PropTypes.string
+  textSendShortcut: PropTypes.string,
+  textDisconnect: PropTypes.string,
+
+  insertToolbarTo: PropTypes.string
```

Comment:
When would this be used? I don't see it being used here at all. 
These propTypes should all be commented.

Reply:
> When would this be used? I don't see it being used here at all.

It is used in VncActions.

https://github.com/patternfly/patternfly-react/blob/223e582264d215dd6341bd87507a2774b4b0e4b0/packages/patternfly-3/react-console/src/VncConsole/VncActions.js#L30-L35

Reply:
> These propTypes should all be commented.

Added comment.